misc/python/materialize/test_analytics/data/base_data_storage.py

```diff
@@ -22,7 +23,7 @@ def __init__(self, database_connector: DatabaseConnector):
     def query_data(
         self,
         query: str,
-        verbose: bool = False,
+        verbose: bool = True,
```

Comment:
Intentional?

Reply:
Yes (https://github.com/MaterializeInc/materialize/commit/a02243ed844329e39bd653ecc8486ed6b1bc074a), to gain some insights about durations.